ui/lib/core/addon/templates/components/replication-table-rows.hbs

```diff
@@ -1,7 +1,8 @@
 {{#unless data}}
+{{!-- ARG TODO test, I don't think empty state is showing --}}
```

Comment:
since this PR is focused on error states, i think it'd be a good idea to verify that the empty state is working here :)